File: CustomerApp/app/src/main/java/com/example/myapplication/ui/fragment/register/SignUpFragment.java
package com.example.myapplication.ui.fragment.register;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.example.myapplication.R;

public class SignUpFragment extends Fragment {

    private EditText username, mobile, email, password, repassword, address;
    private Button btn_signUp;
    private TextView toSignIn;

    private View view;

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        view = inflater.inflate(R.layout.fragment_sign_up, container, false);

        return view;
    }
}

File: CustomerApp/app/src/main/java/com/example/myapplication/ui/fragment/register/SignUpFragment.kt
package com.example.myapplication.ui.fragment.register

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.example.myapplication.databinding.FragmentSignUpBinding
import com.example.myapplication.util.android.base.BaseFragment

class SignUpFragment private constructor() :
    BaseFragment<FragmentSignUpBinding?>() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = FragmentSignUpBinding.inflate(
            inflater,
            container,
            false
        )
        return binding!!.getRoot()
    }

    companion object {
        fun newInstance(): SignUpFragment {
            return SignUpFragment()
        }
    }
}